website/docs/guides/container_service_extension_4_0_cluster_management.html.markdown

```diff
@@ -0,0 +1,272 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: Container Service Extension v4.0 Kubernetes clusters management"
+sidebar_current: "docs-vcd-guides-cse-4-0-cluster-management"
+description: |-
+  Provides guidance on provisioning Kubernetes clusters using Container Service Extension v4.0
+---
+
+# Container Service Extension v4.0 Kubernetes clusters management
+
+## About
+
+This guide explains how to create, update and delete TKGm clusters in a VCD appliance with Container Service Extension v4.0
+installed, using Terraform.
+
+We will use the [`vcd_rde`][rde] resource for this purpose.
+
+~> This section assumes that your CSE installation was done following the [CSE v4.0 installation guide][cse_install_guide].
+That is, CSE Server should be up and running and all elements must be working.
+
+## Pre-requisites
+
+-> Please read also the pre-requisites section in the [CSE documentation][cse_docs].
+
+In order to complete the steps described in this guide, please be aware:
+
+* CSE v4.0 must be installed. Read the [installation guide][cse_install_guide] for more information.
+* Terraform provider needs to be v3.9.0 or above.
+* CSE Server must be up and running.
+
+## Creating a Kubernetes cluster
+
+-> You can have a look at a working example of a TKGm cluster [here][cluster]. It is encouraged to read the following
+section to understand how it works.
+
+To be able to create a TKGm cluster, one needs to prepare a [`vcd_rde`][rde] resource. In the [proposed example][cluster],
+this RDE is named `k8s_cluster_instance`. The important arguments to take into account are:
```

Comment:
Just for clarity, does this mean only the CSE server should be running? What about the Kubernetes plugin created cluster?

Reply:
Yep, only the CSE Server. The UI plugin is optional (it is probably recommended, I could add that note).